utils/githooks/pre-commit.d/20-isort.sh

```diff
@@ -4,6 +4,13 @@
 
 set -ue
 
+# Disable isort check because it doesn't work
+# - it needs to run with -rc
+# - it conflicts with pylint checks
```

Comment:
Interesting ... curious how my PR passed CI pylint check?

Reply:
It seems your PR ran with a different config/env. When I run `isort` nd update all files, it removes some `pylint: disable` lines, which causes the pylint check to fail

Reply:
Should be due to isort version. I'm running 5.9:

```
(ai-1775)yinsun@s07ln0001:~/src/daos$ isort --version

                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.9.0
```

Reply:
Ah! That version doesn't report errors for me. Maybe we should just pin `isort==5.9` in `cq/requirements.txt` so anyone using the githooks gets the right version.
We've ran into similar issues with pylint and flake where we see "new" issues with different versions

Reply:
For the fresh setup, they will pull the latest version anyway. The issue is for existing env we don't have a way to force the contributors to upgrade.

Reply:
> For the fresh setup, they will pull the latest version anyway

For whatever reason, my setup initially pulled in a (cached?) 4.x version. I had to specifically request 5.x+

Reply:
Yeah might be from local cache I guess.